packages/storybook/stories/menu-button.mdx

```diff
@@ -0,0 +1,52 @@
+import { MenuButton } from '@equisoft/design-elements-react';
+import { ArgTypes, Meta, Stories, Story, Canvas } from '@storybook/blocks';
+import * as MenuButtonStories  from './menu-button.stories';
+
+<Meta of={MenuButtonStories} />
+
+# Menu Button
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+
+## Definition
+Menu Button allows users to open a menu with a list of interactive options (actions) [1].
+<Canvas of={MenuButtonStories.Secondary} />
+
+## Usage
+### When to use 
+- Use for actions that will change the application state (either front or back end) [2].
```

Comment:
Ici dans l'article que tu donnes en référence, ils font la différence entre un `button` et un `link`. tu pourrais donc simplifier: "_Use to display a list of action (buttons)_"